pages/docs/couple-your-code/couple-your-code-fem-meshes.md

```diff
@@ -2,7 +2,37 @@
+
+However, this approach runs into trouble in case the support points live on element faces in a discontinuous solution space (discontinuous Galerkin approach), since you define interface nodes multiple times. In particular, the duplicate definition becomes problematic if an interpolant is constructed from the coupling mesh in preCICE, which happens for the `from` mesh in a `read-consistent` setup and for the `to` mesh in a `write-conservative` setup (see also the [mapping configuration](https://precice.org/configuration-mapping.html#restrictions-for-parallel-participants)).
+
+Another issue using higher-order support points might appear in case you run very large cases and strive for the RBF mapping of preCICE. In case the support points are non-equally distributed within the element (shown in our example element above and as it is usually the case in order to preserve a reasonable conditioning of other system matrices), the RBF mapping becomes ill-conditioned due to the varying distance of interface nodes all over the place. Have a look at the paper ["Radial basis function interpolation for black-box multi-physics simulations"](https://upcommons.upc.edu/handle/2117/190255) for detailed information. Again, this issue is only relevant if the RBF interpolant is constructed from the non-equally distributed FEM mesh (see the section above for the related mapping configuration).
+
+## Quadrature points
+
+Quadrature points refer to the points used for numerical quadrature when assembling the matrices of the finite-element system. The location of the quadrature points depend on the employed quadrature formula, where the [Gauss-Legendre quadrature](https://en.wikipedia.org/wiki/Gauss%E2%80%93Legendre_quadrature) is by far the most common choice. Still, there are good reasons to select a different quadrature formula. Using the Gauss-Legendre quadrature points comes in the first place with the same issue of potentially ill-conditioned RBF mappings as described in the section above. However, reading data at the quadrature point location removes the additional mapping from the support point location to the quadrature points, which is usually executed by your solver.
+
+## Conclusion
+
+There are various options how to deal with FEM meshes in preCICE and the best choice depends on your application. In many FEM solvers, the desired write data location is typically related to the support points and the desired read data location is typically related to the quadrature points, since the data is computed/required in these locations. Therefore, you might even consider to define different meshes for reading data from and writing data to preCICE.
```

Comment:
```suggestion
There are various options how to deal with FEM meshes in preCICE and the best choice depends on the solver you want to couple. In many FEM solvers, the desired write data location is typically related to the support points and the desired read data location is typically related to the quadrature points, since the data is computed/required in these locations. Therefore, you might even consider to define different meshes for reading data from and writing data to preCICE.
```